src/sass/_card.scss

```diff
@@ -8,6 +8,28 @@
     padding: 1em;
   }
 
+  .card-image {
+    background-size: 100% 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 100%;
+    @media screen and (max-width: $mobile) {
+      height: 11em;
```

Comment:
we can actually make this height at 10 em for all viewports and it looks good! 😄